### PR TITLE
Add esmlab-regrid recipe

### PR DESCRIPTION
**Pull request by andersy005**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/esmlab-regrid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  host:
+    - python >3.5
+    - pip
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    
+
+  run:
+    - python >3.5
```
Comment (xhochy) [inline recipes/esmlab-regrid/meta.yaml]: Should probably be `>= 3.6`
```suggestion
    - python >=3.6
```

File: recipes/esmlab-regrid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a6492e1f19d52b8631b6e38809ea67a2005ca7b9ca9e56c64022541927f444d
+  
+
+build: 
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py2k or win]
```
Comment (xhochy) [inline recipes/esmlab-regrid/meta.yaml]: The package seems to be a pure Python package, so this could be `noarch`
```suggestion
  noarch: python
```

Comment (andersy005) [reply recipes/esmlab-regrid/meta.yaml]: @xhochy, thank you for your feedback. That's right. The package is a pure Python package but one of its dependency, namely [esmpy](https://anaconda.org/conda-forge/esmpy) is not supported on windows. How should I skip building for Windows?

**Comment by andersy005** [comment]
Closing this for now... The `esmlab-regrid` package development has stalled for the last 10 months. Will submit another PR at another time.

**Pull request closed without merging**